lochness/rpms/__init__.py

```diff
@@ -19,27 +20,59 @@
 logger = logging.getLogger(__name__)
 
 
-def get_rpms_database(rpms_root_path) -> Dict[str, pd.DataFrame]:
+def get_rpms_database(rpms_root_path: str) -> Dict[str, pd.DataFrame]:
     '''Return dictionary of RPMS database in pandas dataframes
 
+    Base on the date in the file name, the most recent csv file among the csv
```

Comment:
TYPO: Based

What is `backed stored`?

Reply:
Thanks for reviewing this. I've corrected the typos and re-written the comments in 575bf0e